src/lavinmq/amqp/queue/queue.cr

```diff
@@ -252,34 +253,56 @@ module LavinMQ::AMQP
     end
 
     private def handle_arguments
+      @effective_args = Array(String).new
```

Comment:
can `handle_arguments` be called multiple times throughout the lifetime of the Queue or why is the array reassigned?

Reply:
Yes, `handle_arguments` is called every time `apply_policy` runs.